services/vendors/j2cli_customization.py

```diff
@@ -0,0 +1,11 @@
+def _generate_vendors_manual_traefik_rule(domains: str, subdomain_prefix: str) -> str:
+    domain_list = domains.strip().strip(",").split(",")
```

Comment:
Can we add assertions here, or even pydantic checks? Would be nice

Reply:
What do you mean? I am not sure I get the idea

Reply:
as we discussed, it is a good idea to implement config validation. We can look into it later :+1: